CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ set(CMAKE_C_STANDARD 99)
 #     Increment MINOR. Set PATCH to 0
 #   If the source code was changed, but there were no interface changes:
 #     Increment PATCH.
-set(LIBRARY_VERSION_MAJOR 13)
+set(LIBRARY_VERSION_MAJOR 14)
```

Comment:
`LIBRARY_VERSION_MAJOR` was bumped because a new `allowIncremental` member was added to the `avifDecoder` struct and a new `isFloat `member was added to the `avifRGBImage` struct.